bugalink-frontend/pages/index.tsx

```diff
@@ -43,7 +45,7 @@ export default function Home() {
             className="aspect-square h-14"
             href={NEXT_ROUTES.PROFILE(user?.user_id)}
           >
-            {user ? <Avatar src={user.photo} /> : <AvatarSkeleton />}
+            {user ? <Avatar src={userPhoto?.photo} /> : <AvatarSkeleton />}
```

Comment:
I believe you are doing this to solve the problem of an outdated JWT token when updating the profile picture, so you call the API every time to bring the user's current photo

Reply:
Makes sense. However, we have the backend URL available as an environment variable (`process.env.NEXT_PUBLIC_BACKEND_URL`) so we can use it to concatenate it with the given `photo` variable and avoid making the unnecessary API call